common.blocks/progressbar/progressbar.js

```diff
@@ -24,7 +24,9 @@ provide(BEMDOM.decl(this.name, /** @lends progressbar.prototype */{
      * @returns {progressbar} this
      */
     setVal : function(val) {
-        this.elem('bar').css('width', (this._val = val) + '%');
+        this.elem('bar')
+            .css('width', (this._val = val) + '%')
+            .attr('aria-valuenow', this._val);
```

Comment:
You add `%` in templates and lose it here. This is inconsistency. In addition, JAWS doesn't add "percent" automatically, so we are constrained to deviate from specification.